tools/cellprofiler/common.xml

```diff
@@ -0,0 +1,445 @@
+<tool id="cp_common" name="Cellprofiler common start modules" version="@CP_VERSION@">

```

Comment:
are we sure that this should be just one tool?

Reply:
Um...If all pipeline starts with these four modules and follow the same order, I don't see why we can't group them together.

Also, We need the header of the pipeline created somewhere in the module, group them as one tool would be easier for user..and us ...